app/assets/javascripts/selectors/columns.js

```diff
@@ -15,9 +15,9 @@ export const getColumns = createSelector(
   (column, stories, project, pastIterations) => {
     switch (column) {
       case Column.CHILLY_BIN:
-        return stories.filter(story => Column.isChillyBin(story)).sort(comparePosition);
+        return stories.all.filter(story => Column.isChillyBin(story)).sort(comparePosition);
```

Comment:
We should have a selector `fromAll` like we'll have `fromSearch`, what do you think?